packages/office-ui-fabric-react/src/demo/pages/ScreenReaderAlertPage/ScreenReaderAlertPage.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import {
+  ExampleCard,
+  PropertiesTableSet,
+  ComponentPage
+} from '../../components/index';
+
+import { getPageRouteFromState } from '../../utilities/pageroute';
+import { AppState } from '../../components/App/AppState';
+import { IComponentDemoPageProps } from '../../components/ComponentPage/IComponentDemoPageProps';
+import { ScreenReaderAlertBasicExample } from './examples/ScreenReaderAlert.Basic.Example';
+import { ScreenReaderAlertRepeatExample } from './examples/ScreenReaderAlert.Repeat.Example';
+import { ScreenReaderAlertSpinnerExample } from './examples/ScreenReaderAlert.Spinner.Example';
+
+const ScreenReaderAlertBasicExampleCode = require('./examples/ScreenReaderAlert.Basic.Example.tsx');
+const ScreenReaderAlertRepeatExampleCode = require('./examples/ScreenReaderAlert.Repeat.Example.tsx');
+const ScreenReaderAlertSpinnerExampleCode = require('./examples/ScreenReaderAlert.Spinner.Example.tsx');
+
+export class ScreenReaderAlertPage extends React.Component<IComponentDemoPageProps, any> {
+  private _url: string;
+
+  constructor() {
+    super();
+    this._url = getPageRouteFromState(AppState, 'UTILITIES', 'ScreenReaderAlert');
+  }
+
+  public render() {
+    return (
+      <ComponentPage
+        title='ScreenReaderAlert'
+        componentName='ScreenReaderAlertExample'
+        exampleCards={
+          <div>
+            <ExampleCard title='ScreenReaderAlert basic' code={ ScreenReaderAlertBasicExampleCode }>
+              <ScreenReaderAlertBasicExample />
+            </ExampleCard>
+            <ExampleCard title='ScreenReaderAlert repeat same message' code={ ScreenReaderAlertRepeatExampleCode }>
+              <ScreenReaderAlertRepeatExample />
+            </ExampleCard>
+            <ExampleCard title='ScreenReaderAlert with Spinner' code={ ScreenReaderAlertSpinnerExampleCode }>
+              <ScreenReaderAlertSpinnerExample />
+            </ExampleCard>
+          </div>
+        }
+        propertiesTables={
+          <PropertiesTableSet componentName='ScreenReaderAlert' />
+        }
+        overview={
+          <div>
+            <p>
+              ScreenReaderAlerts are used to read out the notifications on the page to the blind people with the help of screen readers like JAWS, Chrome VOX and NVDA. It is implemented by live region and overcomes some known issues about live region when using with React.
```

Comment:
I tested with NVDA&Firefox, Narrator&Edge. Add support screen readers to the comment as well as the demo page. Thanks